tests/functional/nfs_ganesha/root-squash/test_nfs_ganesha_root_squash_mount_unmount.py

```diff
@@ -0,0 +1,219 @@
+                       "file created by nobody user")
+
+        # Create some more files on mount point post remount.
+        cmd = ("for i in {1..20}; do touch %s/remount_file$i; done"
+               % self.mounts[0].mountpoint)
+        ret, _, err = g.run(self.mounts[0].client_system, cmd,
+                            user=self.mounts[0].user)
+        self.assertEqual(ret, 0, err)
+
+        # Get stat of all the files/dirs created.
+        ret = get_mounts_stat(self.mounts)
+        self.assertTrue(ret, "Stat failed on some of the clients")
+        g.log.info("Successful in getting stats of files/dirs "
+                   "from mount point")
+
+        # Check for owner and group of all the files
+        for mount_obj in self.mounts:
+            for i in range(1, 21):
+                cmd = ("ls -l %s/remount_file%i | awk '{ print $3, $4 }' |sort"
+                       % (mount_obj.mountpoint, i))
+                ret, out, err = g.run(mount_obj.client_system, cmd)
+                self.assertFalse(ret, err)
+        if is_rhel7:
+                self.assertIn("nobody nobody", out,
+                              "Owner and group is not nobody")
+        else:
+                self.assertIn("nfsnobody nfsnobody", out,
+                              "Owner and group is not nobody")
+        g.log.info("Owner and group of file is nobody")
+
+        # Disable root-squash
+        ret = set_root_squash(self.mnode, self.volname, squash=False,
+                              do_refresh_config=True)
+        self.assertTrue(ret, "Failed to disable root-squash on volume")
+        g.log.info("root-squash is disable on the volume")
+
+        # Edit file created by root user
+        for mount_obj in self.mounts:
+            ret = append_string_to_file(mount_obj.client_system,
+                                        "%s/dir15/file15"
+                                        % mount_obj.mountpoint,
+                                        'hello')
+            self.assertTrue(ret, "Unexpected:root user should be allowed to "
+                                 "edit the file created by root user")
+            g.log.info("Successful:root user successful in editing file "
+                       "created by root user")
+
+    def tearDown(self):
+
+        # Unexport volume
+        unexport_nfs_ganesha_volume(self.mnode, self.volname)
+        ret = wait_for_nfs_ganesha_volume_to_get_unexported(self.mnode,
+                                                            self.volname)
+        if not ret:
+            raise ExecutionError("Failed:Volume %s is not unexported."
+                                 % self.volname)
+        g.log.info("Unexporting of volume is successful")
+
+        # Unmount and cleanup Volume
+        ret = self.unmount_volume_and_cleanup_volume(self.mounts)
+        if ret:
+            g.log.info("Successful unmount and cleanup of volume")
+        else:
```

Comment:
Do we really need this or can we do it similar to what we did in the above case?